src/actions/config.js

```diff
@@ -26,33 +25,45 @@ export function configFetchError() {
   };
 }
 
-export function asyncConfigFetch() {
+export function asyncConfigInit() {
   return dispatch => {
-    dispatch(configFetch());
+    dispatch(asyncConfigFetch('./config.js'));
+    dispatch(asyncConfigFetch('./userConfig.js'));
```

Comment:
Javascript is single threaded but you're right its possible the `userConfig.js` request finishes before `config.js` even if its dispatched second.  I'll update the code to enforce the ordering.